enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -133,6 +136,16 @@ const (
 
 	// The path in the guest where VFS is mounted.
 	guestVFSMountDir = "/vfs"
+
+	// How much of the context deadline to allocate towards actually executing
+	// the command. The remaining fraction is dedicated allocated towards
+	// collecting outputs.
+	execDeadlineFraction = 0.85
```

Comment:
Seems like a small fixed timeout would be a little simpler to calcuate / work with -- is a second or two enough time to budget for collecting outputs?

Reply:
Resolved offline -- it's probably less surprising to the user if we use a 1s timeout (changed to 1s).